src/pymor/models/transforms.py

```diff
@@ -0,0 +1,171 @@
+        assert coefficients[0]*coefficients[3] != coefficients[1]*coefficients[2]
+
+        if normalize:
+            coefficients = coefficients / np.sqrt(coefficients[0]*coefficients[3]-coefficients[1]*coefficients[2])
+            coefficients *= np.sign(coefficients[0])
+
+        self.__auto_init(locals())
+
+    @classmethod
+    def from_points(cls, w, z=(0, 1, np.inf), name=None):
+        """Constructs a Moebius transformation from three points and their images.
+
+        A Moebius transformation is completely determined by the images of three distinct points on
+        the Riemann sphere under transformation.
+
+        Parameters
+        ----------
+        w
+            A tuple, list or |NumPy array| of three complex numbers that are transformed.
+        z
+            A tuple, list or |NumPy array| of three complex numbers represent the images of `w`.
+            Defaults to `(0, 1, np.inf)`.
+        name
+            Name of the transformation.
+
+        Returns
+        -------
+        M
+            The corresponding |MoebiusTransformation|.
+        """
+        assert len(z) == 3
+        assert len(w) == 3
+
+        A = np.zeros([3, 4], dtype=complex)
+        for i in range(3):
+            if np.isinf(w[i]) and np.isinf(z[i]):
+                A[i] = [0, 0, -1, 0]
+            elif np.isinf(w[i]):
+                A[i] = [0, 0, -z[i], -1]
+            elif np.isinf(z[i]):
+                A[i] = [1, 0, -w[i], 0]
+            else:
+                A[i] = [z[i], 1, -w[i]*z[i], -w[i]]
+
+        return cls(null_space(A).ravel(), name=name)
+
+    def inverse(self, normalize=False):
+        """Returns the inverse Moebius transformation by applying the inversion formula.
+
+        Parameters
+        ----------
+        normalize
+            If `True`, the coefficients are normalized, i.e. :math:`ad-bc=1`. Defaults to `False`.
+
+        Returns
+        -------
+        M
+            The inverse |MoebiusTransformation|.
+        """
+        a, b, c, d = self.coefficients
+        coefficients = np.array([d, -b, -c, a])
+        return MoebiusTransformation(coefficients, normalize=normalize, name=self.name + '_inverse')
+
+    def _mapping(self, x):
+        a, b, c, d = self.coefficients
+        if c != 0 and np.allclose(x, -d / c):
+            return np.inf
+        elif c != 0 and np.isinf(x):
+            return a / c
+        elif c == 0 and np.isinf(x):
+            return np.inf
+        else:
+            return (a * x + b) / (c * x + d)
+
+    def __call__(self, x):
+        x = np.squeeze(np.array(x))
+        assert x.ndim <= 1
+        return np.vectorize(self._mapping)(x)
+
+    def __matmul__(self, other):
+        assert isinstance(other, MoebiusTransformation)
+        M = self.coefficients.reshape(2, 2) @ other.coefficients.reshape(2, 2)
+        return MoebiusTransformation(M.ravel())
+
+    def __str__(self):
+        return (
+            f'{self.name}: C^1 --> C^1\n'
```

Comment:
Maybe Unicode works...

```suggestion
            f'{self.name}: ℂ --> ℂ\n'
```